### PR TITLE
refactor: change page structure, adapt tests (DEV-1725)

### PR DESCRIPTION
**Pull request by Vijeinath**
resolves DEV-1725

### CODE REVIEW

File: src/app/services/project-repository.service.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import {Injectable} from "@angular/core";
 import {CpeApiService, iPage, iProject} from "./cpe-api.service";
 import {Page, Project, Widget} from "../cpe/model/page-data-structure";
-import {Observable, forkJoin, map, mergeMap, of, from, toArray} from "rxjs";
+import {Observable, forkJoin, map, mergeMap, of, from, toArray, tap} from "rxjs";
```
Comment (EricSommerhalder) [inline src/app/services/project-repository.service.ts:4]: accidental import "tap"?

**Review by EricSommerhalder — CHANGES_REQUESTED** [review]
VERY minor comment, otherwise it looks very good! Thanks a lot!

File: src/app/services/project-repository.service.ts (hunk 1 above)
Comment (Vijeinath) [reply src/app/services/project-repository.service.ts]: Done in https://github.com/dasch-swiss/dsp-cpe/pull/17/commits/d139bdf39a1594bf7cfcfd4d68e54ee589d5a9d3

**Review by EricSommerhalder — APPROVED** [review]
Thanks alot

**Pull request merged**